### PR TITLE
CVE's Merge Forward into Master

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Comment by s0undt3ch** [comment]
re-run full all

**Review by twangboy — DISMISSED** [review]

**Review by dwoz — DISMISSED** [review]

**Review by Ch3LL — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,52 @@ Versions are `MAJOR.PATCH`.
 
 # Changelog
 
+Salt 3002.4 (2021-02-09)
```
Comment (frogunder) [inline CHANGELOG.md:10]: Not sure how this line `Salt 3002.4 (2021-02-09)` got here, but it should be deleted.

**Review by frogunder — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (s0undt3ch) [reply CHANGELOG.md]: Removed

**Comment by s0undt3ch** [comment]
Pushed the changelog fix and the test were passing before that. It also already had 3 approvals. Merging.

**Comment by s0undt3ch** [comment]
Hmm, don't know why, but I can't merge it

**Pull request merged**